ui/widget.js

```diff
@@ -514,6 +514,16 @@ $.Widget.prototype = {
 			}
 		}
 
+		this._on( options.element, {
+			"remove": function( event ) {
+				$.each( that.classesElementLookup, function( key, value ) {
+					if ( $.inArray( event.target ) ) {
```

Comment:
Do you happen to know the performance of `.not()` vs `$.inArray()`? Is this likely to actually improve performance?

cc @jquery/core